package.json

```diff
@@ -23,7 +23,10 @@
   },
   "dependencies": {
     "after": "^0.8.1",
+    "browserify": "^13.0.0",
+    "browserify-shim": "^3.8.12",
```

Comment:
you've required browserify-shim. Why haven't you used the `"browser"` entry to replace request?